GooglePlacesAutocomplete.js

```diff
@@ -767,6 +772,7 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
     ) {
       return (
         <FlatList
+          {...props}
```

Comment:
Should we give this much flexibility to the user?
Cause we are using custom function, and overriding that might break other functionalities.
And if we haven't created custom function for any other prop, it will be passed without any problems either way.

Reply:
it's important to be able to overwrite `renderItem` for exemple

Reply:
Shouldn't we do this safely via prop?
But if you want, I will readjust it to the bottom again.

Reply:
Done

Reply:
Thank you @shubham1206agra